framework/aster-frame/Cargo.toml

```diff
@@ -30,7 +30,8 @@ pod = { git = "https://github.com/asterinas/pod", rev = "d7dba56" }
 spin = "0.9.4"
 static_assertions = "1.1.0"
 tdx-guest = { version = "0.1.0", optional = true }
-trapframe = { git = "https://github.com/asterinas/trapframe-rs", rev = "2f37590" }
+# trapframe = { git = "https://github.com/asterinas/trapframe-rs", rev = "2f37590" }
+trapframe = { path = "../../../trapframe-rs" }
```

Comment:
So you need [PR #3](https://github.com/asterinas/trapframe-rs/pull/3) to be accepted?